### PR TITLE
Depth estimation task guide

### PR DESCRIPTION
**Pull request by MKhalusova**
This PR adds a zero-shot depth estimation task guide that covers inference with a pipeline, as well as manually, for monocular depth estimation supported by DPT and GLPN.

### CODE REVIEW

**Comment by MKhalusova** [comment]
PR with images: https://huggingface.co/datasets/huggingface/documentation-images/discussions/64

File: docs/source/en/model_doc/dpt.mdx [hunk 1]
```diff
@@ -34,6 +34,7 @@ A list of official Hugging Face and community (indicated by 🌎) resources to h
 
 - Demo notebooks for [`DPTForDepthEstimation`] can be found [here](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/DPT).
 - See also: [Semantic segmentation task guide](./tasks/semantic_segmentation)
+- [Monocular depth estimation task guide](./tasks/monocular_depth_estimation.mdx)
```
Comment (amyeroberts) [inline docs/source/en/model_doc/dpt.mdx:37]: ultra ultra nit: I would remove the `See also: ` in the line above to keep it consistent

**Review by amyeroberts — APPROVED** [review]
LGTM - thanks for adding!

File: docs/source/en/model_doc/dpt.mdx (hunk 1 above)
Comment (MKhalusova) [reply docs/source/en/model_doc/dpt.mdx:37]: Done! :)

File: docs/source/en/model_doc/dpt.mdx [hunk 2]
```diff
@@ -33,7 +33,8 @@ This model was contributed by [nielsr](https://huggingface.co/nielsr). The origi
 A list of official Hugging Face and community (indicated by 🌎) resources to help you get started with DPT.
 
 - Demo notebooks for [`DPTForDepthEstimation`] can be found [here](https://github.com/NielsRogge/Transformers-Tutorials/tree/master/DPT).
-- See also: [Semantic segmentation task guide](./tasks/semantic_segmentation)
+- [Semantic segmentation task guide](../tasks/semantic_segmentation)
+- [Monocular depth estimation task guide](../tasks/monocular_depth_estimation.mdx)
```
Comment (amyeroberts) [inline docs/source/en/model_doc/dpt.mdx:37]: @MKhalusova - I've just noticed that these links shouldn't have the `.mdx` at the end. Could you open another PR to remove them?

Comment (NielsRogge) [reply docs/source/en/model_doc/dpt.mdx:37]: Thanks for adding! Noticed images aren't rendered properly, might need to double check those links as well

Comment (NielsRogge) [reply docs/source/en/model_doc/dpt.mdx:37]: Update, have merged the PR on the hub, that's working now 👍

Comment (MKhalusova) [reply docs/source/en/model_doc/dpt.mdx:37]: Thank you @NielsRogge !

**Pull request merged**